godtools/App/Services/Renderer/Views/MobileContent/Views/ContentAnimation/MobileContentAnimationViewModel.swift

```diff
@@ -11,11 +11,21 @@ import GodToolsToolParser
 
 class MobileContentAnimationViewModel: MobileContentViewModel {
     
+    enum PlaybackState {
+        case pause
+        case play
+        case stop
+    }
+    
+    private static let eventToggleAnimation: String = "toggleAnimation"
```

Comment:
the `toggleAnimation` event id is just an arbitrary event id I created, in my sample content XML that event id is on both the playListeners and stopListeners and will trigger the corresponding listener based on if the animation is currently playing or not

Reply:
all that to say you shouldn't have a random event id hard-coded like this

Reply:
Hey Daniel, I'm not crazy about it either.  I'll need to update the ViewModel to have a two-way binding so it knows when the animation is truly stopped.  Currently tracking the state only in the ViewModel which won't get updated when an animation finishes.

Reply:
Gonna work on adding that in.